content/docs/tasks/security/authz-tcp/index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Authorization for TCP services
+description: Shows how to set up role-based access control for TCP services in the mesh.
+weight: 40
+keywords: [security,access-control,rbac,tcp,authorization]
```

Comment:
Same question here. Do we use "tcp" keyword anywhere else?

Reply:
Yes, like `content/blog/2018/egress-tcp/index.md` and `content/docs/tasks/telemetry/tcp-metrics/index.md`